gradle/libs.versions.toml

```diff
@@ -178,6 +182,7 @@ jsonTest = { module = "org.json:json", version.ref = "jsonTest" }
 junit = { module = "junit:junit", version.ref = "junit" }
 kaspresso = { module = "com.kaspersky.android-components:kaspresso", version.ref = "kaspresso" }
 koin-android = { module = "io.insert-koin:koin-android", version.ref = "koinAndroid" }
+koin-androidx-compose = { module = "io.insert-koin:koin-androidx-compose", version.ref = "koinAndroidCompose" }
```

Comment:
*Might* be a good idea not to use this to not be able to resolve things with a `get()` inside composables, but force us to resolve these higher up in the activity, in the ViewModel or something like that. Because it can be a pretty uncommon thing to resolve say a UseCase inside a composable. But we'll see if we have valid use cases for it and if we want to keep this.